templates/CRM/common/drupal6.tpl

```diff
@@ -23,57 +23,4 @@
  | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
  +--------------------------------------------------------------------+
 *}
-{if $config->debug}
-{include file="CRM/common/debug.tpl"}
-{/if}
-
-<div id="crm-container" class="crm-container{if $urlIsPublic} crm-public{/if}" lang="{$config->lcMessages|truncate:2:"":true}" xml:lang="{$config->lcMessages|truncate:2:"":true}">
-
-{crmNavigationMenu is_default=1}
-
-{if isset($browserPrint) and $browserPrint}
-{* Javascript window.print link. Used for public pages where we can't do printer-friendly view. *}
-<div id="printer-friendly">
-<a href="#" onclick="window.print(); return false;" title="{ts}Print this page.{/ts}">
-  <i class="crm-i fa-print"></i>
-</a>
-</div>
-{else}
-{* Printer friendly link/icon. *}
-<div id="printer-friendly">
-<a href="{$printerFriendly}" target='_blank' title="{ts}Printer-friendly view of this page.{/ts}">
-  <i class="crm-i fa-print"></i>
-</a>
-</div>
-{/if}
-
-{*{include file="CRM/common/langSwitch.tpl"}*}
```

Comment:
So this line is removed from the final tpl - but as it is commented out that seems fine